internal/pkg/cli/svc_deploy.go

```diff
@@ -458,14 +458,16 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 }
 
 func (o *deploySvcOpts) validateAppVersion(svc *manifest.LoadBalancedWebService) error {
-	appVersion, err := o.appVersionGetter.Version()
-	if err != nil {
-		return fmt.Errorf("get version for app %s: %w", o.appName, err)
-	}
+	var appVersion string
+	var err error
 	if aws.StringValue(svc.Alias) != "" && o.targetApp.RequiresDNSDelegation() {
+		appVersion, err = o.appVersionGetter.Version()
```

Comment:
```suggestion
	if aws.StringValue(svc.Alias) != "" && o.targetApp.RequiresDNSDelegation() {
		appVersion, err := o.appVersionGetter.Version()
```

Reply:
That's for future use actually since we don't need to get the app version again for the other field validation if alias is enabled.